### PR TITLE
Add minor features to debugger html client

### PR DESCRIPTION
**Pull request by polaroi8d**
- disable resizable textarea
- block to write the log are
- add feature which auto scroll the log to the bottom, for easier readability

JerryScript-DCO-1.0-Signed-off-by: Levente Orban orbanl@inf.u-szeged.hu

### CODE REVIEW

File: jerry-debugger/jerry-client-ws.html [hunk 1]
```diff
@@ -2,31 +2,44 @@
...
-
-<textarea id="log" rows="20" cols="80"></textarea><br>
-
-Getting help: type 'help' in the command line below.<br>
-
-<input id="command" type="text" onkeypress="debuggerCommand(event); return true;">
+<div class="container">
+  <div class="form-group">
+    <textarea readonly class="form-control center-block" rows="16" id="log"></textarea>
+    </div>
```
Comment (LaszloLango) [inline jerry-debugger/jerry-client-ws.html]: wrong indentation

File: jerry-debugger/jerry-client-ws.html [hunk 2]
```diff
@@ -2,31 +2,45 @@
 <html>
 <head>
 <meta charset="UTF-8">
+<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```
Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.html]: I think the page should work without network connection.

Comment (polaroi8d) [inline jerry-debugger/jerry-client-ws.html]: Hmmm, good point. Then there are two possibility, upload the minified boostrap files ( imho bad solution ) or just delete the bootstrap. What do you think?

Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.html]: I would prefer delete. If something is needed, just add it to the source code of the page.

**Comment by polaroi8d** [comment]
Deleted the Boostrap CND. Updated the PR

**Pull request merged**